libs/shared/constants/src/lib/projectBasePath.ts

```diff
@@ -0,0 +1,7 @@
+export enum ProjectBasePath {
+  ServicePortal = '/minarsidur',
+  ConsultationPortal = '/samradsgatt',
+  ApplicationSystem = '/umsoknir',
+  AirDiscountScheme = '/loftbru',
+  AdminPortal = '/stjornbord',
+}
```

Comment:
We should add `CarRecycling = '/app/skilavottord`` for skilavottord-web

Reply:
Done